rpc/cache/service.pb.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.27.1
+// 	protoc-gen-go v1.34.0
```

Comment:
looks like we can skip this change

Reply:
this change results the versions of `protoc-gen-go` for `cache/` and `common` to the same version.  it was build automatically

Reply:
removed this update for this PR)